go.mod

```diff
@@ -32,6 +32,7 @@ require (
 	atomicgo.dev/cursor v0.2.0 // indirect
 	atomicgo.dev/keyboard v0.2.9 // indirect
 	atomicgo.dev/schedule v0.1.0 // indirect
+	github.com/NVIDIA/go-nvml v0.12.4-1
```

Comment:
Please move this entry to the first list, as the second list only contains indirect dependencies.
Or was this intentional?

Reply:
I used VS Code to write this, and it somehow sorts the imports automatically whenever I save. That's also why some of your imports have been reordered